src/Tools/ExternalAccess/OmniSharp/Completion/OmniSharpCompletionOptions.cs

```diff
@@ -10,6 +10,10 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Completion
         bool ShowItemsFromUnimportedNamespaces)
     {
         internal CompletionOptions ToCompletionOptions()
-            => CompletionOptions.Default with { ShowItemsFromUnimportedNamespaces = ShowItemsFromUnimportedNamespaces };
+            => CompletionOptions.Default with
+            {
+                ShowItemsFromUnimportedNamespaces = ShowItemsFromUnimportedNamespaces,
+                UpdateImportCompletionCacheInBackground = true
```

Comment:
@JoeRobich @333fred FYI, I have set the O# default behavior same as VS (try to refresh cache in the background when completion is being triggered but it's not in the right context for import completion). Note the background work might make some test behavior nondeterministic but you can use `ForceExpandedCompletionIndexCreation` (which I think we will need to expose to O#)